python/pylibcudf/pylibcudf/strings/findall.pyx

```diff
@@ -38,3 +38,35 @@ cpdef Column findall(Column input, RegexProgram pattern):
         )
 
     return Column.from_libcudf(move(c_result))
+
+
+cpdef Column find_re(Column input, RegexProgram pattern):
+    """
+    Returns character positions where the pattern first matches
+    the elements in source_strings.
```

Comment:
Suggesting since the argument name is `input` here
```suggestion
    Returns character positions where the pattern first matches
    the elements in input strings.
```